### PR TITLE
PR for Feature/homepage

### PR DESCRIPTION
**Pull request by Lameck1**
This PR is a submission of the following features implemented in the restaurant page project

* A home page
* Menu page
* Contact Page

The app has implementations of modules for the above features. The modules have been exported and imported into main.js and bundled by webpack.

### CODE REVIEW

File: src/assets/js/menu.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import createElem from './createElem';
+
+export default () => {
+  // renders menu section
```
Comment (afanimartin) [inline src/assets/js/menu.js]: - Remove this comment

File: src/assets/js/nav.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import createElem from './createElem';
+
+export default () => {
+  // renders nav section
```
Comment (afanimartin) [inline src/assets/js/nav.js]: - Remove this comment

File: src/assets/js/home.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import createElem from './createElem';
+
+export default () => {
+  // renders home page
```
Comment (afanimartin) [inline src/assets/js/home.js]: - Remove this comment

**Review by afanimartin — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 🙏🏿 🙏🏿 
Nice work on this project. Your app looks quite professional. Well done.
### High lights 🌵 🌵 🌵 
- Readme looks professional
- Correct use of modules
- Required packages added to `package.json` file

### Changes 🙅🏿 🙅🏿 🙅🏿 
- Remove comments from your codebase. Comments make your code look ugly

### Martin Afani
#### Slack: whiz25

**Review by Forison — APPROVED** [review]
## Status::Approved✔️

# 👋 @Lameck1 

#### Good job on addressing the issue raised in the previous review, kindly go ahead and merge the branch and proceed to your next task😇 .

#### Happy coding👨‍💻

Addo Forison

**Pull request merged**